tests/functional/modules/pyi_pkg_resources_provider/package/pyi_pkgres_testpkg/a.py

```diff
@@ -0,0 +1,10 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
```

Comment:
Let's save electric power and bandwidth: These are empty files, no need for a copyright header. A `#`suffices.

Same for the other empty ones.

Reply:
OK, makes sense.